src/components/vault/EntryDetails.js

```diff
@@ -177,17 +208,41 @@ const FieldRow = ({ field, editing, onFieldNameUpdate, onFieldUpdate, onRemoveFi
     ) : (
       field.title || field.property
     );
+  const renderMenu = (
+    <Menu>
+      {/*
+        The following is in the parent level due to:
```

Comment:
Damn

Reply:
ikr.. Oh well, we can check back with it later..